core/src/test/java/org/infinispan/distexec/DistributedExecutorFailoverTest.java

```diff
@@ -124,4 +160,25 @@ public int maxFailoverAttempts() {
          return maxFailoverCount;
       }
    }
+
+   static class SameNodeTaskFailoverPolicy implements DistributedTaskFailoverPolicy {
+      private final int maxFailoverCount;
```

Comment:
Okay. Fixed. Since slaskawi mis-referred the line. The code isn't shown as outdated.
And also I'd like to know whether infinispan with an atomic integer cluster wide?
Hazelcast is with one. But it seems I can't find it in infinispan.

Before development of my project, I take a look at all the IMDG as well. And I chose infinispan in the end. Because it's more familiar with my previous using stuffs from JBoss, like Weld, WildFly, Netty, Resteasy, Hibernate and so on.